libs/editor/WordPressEditor/build.gradle

```diff
@@ -39,7 +36,7 @@ android {
         versionName "1.3"
         minSdkVersion 21
         targetSdkVersion 29
-        buildConfigField "boolean", "BUILD_GUTENBERG_FROM_SOURCE", (rootProject.ext.has("buildGutenbergFromSource") ? rootProject.ext.buildGutenbergFromSource : false).toString()
+        buildConfigField "boolean", "BUILD_GUTENBERG_FROM_SOURCE", buildGutenbergFromSource.toString()
```

Comment:
Does the `BUILD_GUTENBERG_FROM_SOURCE` variable still have a purpose? I would really like to remove it if possible.

Reply:
[This](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java#L535-L537) is why we still need to pass this. If we can rework that somehow, we can remove this config field.

Reply:
Could we just tie this to the same logic that determines if we're doing a local build? In other words, this would be true if and only if we were locally building Gutenberg?

Reply:
@mchowning Do you mean do this from the Gutenberg repo? Otherwise, that's exactly how it currently works. The `buildGutenbergFromSource` is not something developers should modify, it is set depending on whether they have the `localGutenbergMobilePath` property in their `local-builds.gradle` file or not.

Reply:
Ah I missed that 🤦 :

```
def buildGutenbergFromSource = gradle.ext.includedBuildGutenbergMobilePath != null
```

Thanks for explaining.

Reply:
I don't feel strongly, but do you think it would be worth renaming this variable and config field to something like `gutenbergIncludedBuild` and `GUTENBERG_INCLUDED_BUILD` just to make that more obvious?

Reply:
I am not particularly happy with this flag either, let me see if I can read this property from `react-native-bridge` instead.

Reply:
I've reworked this in https://github.com/oguzkocer/gutenberg/commit/d9a4a70ed6deaa29f0a35c55cc6f565259e1c689 and b45beeb29110a9e5dc36aa1d9df8b5bd49a3c029. We no longer need to pass any arguments to `react-native-bridge` as it'll decide whether to include the bundle or not.